taichi/backends/metal/kernel_manager.cpp

```diff
@@ -975,6 +983,7 @@ class KernelManager::Impl {
 
   CompileConfig *const config_;
   const CompiledStructs compiled_structs_;
+  BufferSize buffer_size_data;
```

Comment:
Private memeber variables end with `_`:
 
```suggestion
  BufferSize buffer_size_data_;
```